src/unlimited.h

```diff
@@ -196,6 +201,7 @@ extern CTweak<uint64_t> blockSigopsPerMb;
 extern CTweak<uint64_t> coinbaseReserve;
 extern CTweak<uint64_t> blockMiningSigopsPerMb;
 
+extern std::list<CStatBase*> mallocedStats;
```

Comment:
Structurally it's simpler than a `list`, the addresses are guaranteed to be saved next to each other, could be useful later on if you want to examine the memory. It also removes the need to `#include <list>` which means less work for the compiler.